fs_mark/fs_mark.py

```diff
@@ -0,0 +1,197 @@
+import trunner
+import time
+import os
+import yaml
+import re
+import pexpect
+import timing_data as t_data
+import psh.tools.psh as psh
+
+from datetime import datetime
+from trunner.ctx import TestContext
+from trunner.dut import Dut
+from trunner.types import TestResult, TestSubResult, Status
+
+
+def clean(dut, test_name, ctx: TestContext) -> int:
+    # Dummyfs doesn't need clean since target is rebooted after every test
+    if not ctx.target.rootfs:
+        return 0
+
+    if '-k' in test_name or '-L' in test_name or '-F' in test_name:
+        args = test_name.split()
+        args = args[1:]
+
+        def get_dir(x):
+            if args[x[0] - 1] == '-d':
+                return x[1]
+
+        dirs = list(map(lambda x: get_dir(x), enumerate(args)))
+        dirs = [x for x in dirs if x is not None]
+
+        dut.expect(re.escape(ctx.target.shell_prompt), timeout=120)
+        psh._send(dut, f'/bin/fs_mark_clean {" ".join(dirs)}')
+        try:
+            idx = dut.expect([re.escape(ctx.target.shell_prompt),
+                              r".+?Error in remove_dir_recursive.+?"], timeout=1800)
+
+            if idx == 1:
+                return -1
+            elif idx == 0:
+                return 0
+        except pexpect.TIMEOUT:
+            return -2
+    else:
+        return 0
+
+
+def harness(dut: Dut, ctx: TestContext, result: TestResult):
+    target = trunner.ctx.target.name
+    logs_path = '~/remote/logs/fs_mark_logs_test'
```

Comment:
Hmm, I think it would be much better to analyze times from xml files available on gh. You can simply download multiple files from various nightly campaigns, for more information please contact @maska989 

```suggestion
```

Reply:
Syscall timings will be added later.